File: Dataset/Leetcode/train/75/222.java
class Solution {
    public void XXX(int[] nums) {
        quickSort(nums, 0, nums.length - 1);
    }
    public static void quickSort(int[] nums, int left, int right) {
        if(left > right)
            return;
        int l = left;
        int r = right;
        int key = nums[l];
        while (l < r) {
            while (l < r && nums[r] >= key)
                r--;
            nums[l] = nums[r];
            while (l < r && nums[l] <= key)
                l++;
            nums[r] = nums[l];
        }
        nums[l] = key;
        quickSort(nums, left, l - 1);
        quickSort(nums, r + 1, right);
    }
}

File: Dataset/Leetcode/train/58/222.py
 class Solution:
    def XXX(self, s: str) -> int:
        length=len(s)
        count=0
        i=0
        while length-i-1>=0 and s[length-i-1] ==' ':
            i+=1
        while length-i-1>=0 and s[length-i-1]!=' ':
            count+=1
            i+=1
        return count


